docs/kics-auto-remediation.md

```diff
@@ -0,0 +1,30 @@
+# KICS AUTO REMEDIATION
+
+With this new feature, KICS provides auto remediation for simple replacements and simple additions in a single line.
+
+Note that this feature will be only available for Terraform, for now.
+
+<p align="center">
+<img width="950" alt="image" src="https://user-images.githubusercontent.com/74001161/177953750-3d279868-8cdb-44c9-86f2-379b05bb85d4.png">
+</p>
+
+
+
+## HOW KICS AR WORKS?
+
+
+1. As a first step, you will need to scan your project/file and generate a JSON report.
+
+   Example: ```docker run -v /home/cosmicgirl/:/path/ kics scan -p /path/sample.tf -i "41a38329-d81b-4be4-aef4-55b2615d3282,a9dfec39-a740-4105-bbd6-721ba163c053,2bb13841-7575-439e-8e0a-cccd9ede2fa8" --no-progress -o /path/results --report-formats json```
+   
+   If KICS makes available a remediation for a result, the result will have the fields `remediation` and `remediation_type` defined. As an example, please see:
+   <p align="center">
+   <img width="700" alt="image" src="https://user-images.githubusercontent.com/74001161/177957089-7007d5c0-aea5-4f3a-8300-7008ab0e6312.png">
+   </p>
+   
+   
+2. If your JSON report has any result with remediation, you will need to run the new KICS command: fix. 
+
+   If you want KICS to remediate all the remediation, you can run ```docker run -v /home/cosmicgirl/:/path/ kics fix --results /path/results/results.json -v```.
```

Comment:
If you want KICS to remediate all the remediation
--->
To fix the code with all remediation suggestions,